src/pal/src/misc/error.cpp

```diff
@@ -96,25 +94,8 @@ PALAPI
 GetLastError(
          VOID)
 {
-    DWORD retval;
-    CPalThread *pThread;
-
-    PERF_ENTRY(GetLastError);
-    ENTRY("GetLastError ()\n");
-    
-    pThread = InternalGetCurrentThread();
-    if (pThread == NULL)
-    {
-        retval = StartupLastError;
-        goto done;
-    }
-
-    retval = pThread->GetLastError();
-
-done:
-    LOGEXIT("GetLastError returns %d\n",retval);
-    PERF_EXIT(GetLastError);
-    return retval;
+    // Reuse errno to store the last error
+    return errno;
```

Comment:
The correctness of this seems to depend on CPalThread's Get/SetLastError being implemented as a wrapper for errno.  Should we add a comment to that effect?